backend/Squirrel.Core/Squirrel.Core.WebAPI/Controllers/ProjectController.cs

```diff
@@ -40,10 +40,10 @@ public async Task<ActionResult<ProjectDto>> GetProject(int projectId)
             return Ok(await _projectService.GetProjectAsync(projectId));
         }
 
-        [HttpGet]
-        public async Task<ActionResult<List<ProjectDto>>> GetAllProjects()
+        [HttpGet("createdby/{userId}")]
+        public async Task<ActionResult<List<ProjectDto>>> GetAllUserProjects(int userId)
```

Comment:
Can you remove userId as a parameter please?

Reply:
You can create service to store userId on every request (just for request time) and after that use it where needed (like on getting/creating data)

Reply:
done via service and middleware